src/components/generic/editable-field-components/MetaformHtmlFieldComponent.tsx

```diff
@@ -0,0 +1,87 @@
+import { FormControl, InputLabel, OutlinedInput, WithStyles } from '@material-ui/core';
+import React from 'react';
+import { Metaform, MetaformField } from '../../../generated/client';
+import strings from '../../../localization/strings';
+import styles from "../../../styles/form-edit-screen";
+
+/**
+ * Component props
+ */
+interface Props extends WithStyles<typeof styles> {
+  field: MetaformField;
+  fieldId: string;
+  fieldLabelId: string;
+  metaform: Metaform;
+  fieldName?: string;
+}
+
+/**
+ * Component state
+ */
+interface State {
+  metaform: Metaform;
+}
+
+/**
+ * Component for Metaform memo field
+ */
+export class MetaformHtmlComponent extends React.Component<Props, State> {
+
+  /**
+   * Constructor
+   * 
+   * @param props component props
+   */
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      metaform: this.props.metaform
+    };
```

Comment:
I don't quite get the point having props as local state...
The better way might be having a callback setState passed to this component?
e.g. 
```
setMetaform = (newMetaform: Metaform) => this.setState({ metaform: newMetaform });
```

Reply:
And if it's really necessary, please use destructuring syntax

Reply:
There is no point and it was not actually working as intended, thanks for pointing this out. I'll fix this.